tests/lib/rules/no-unknown-property.js

```diff
@@ -89,6 +89,7 @@ ruleTester.run('no-unknown-property', rule, {
     { code: '<div onPointerDown={this.onDown} onPointerUp={this.onUp} />' },
     { code: '<input type="checkbox" defaultChecked={this.state.checkbox} />' },
     { code: '<div onTouchStart={this.startAnimation} onTouchEnd={this.stopAnimation} onTouchCancel={this.cancel} onTouchMove={this.move} onMouseMoveCapture={this.capture} onTouchCancelCapture={this.log} />' },
+    { code: '<link precedence="medium" href="https://foo.bar" rel="canonical" />' },
```

Comment:
shouldn't this test case only pass when the react version is set to 19+?

Reply:
definitely. i used `react: { version: '19.0.0-rc.0' }`, but if it makes more sense to do `19.0.0` even though that doesn’t match any released versions, happy to do that.